app-accessibility/caribou/caribou-0.4.21-r3.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+PYTHON_REQ_USE="xml"
+
+inherit gnome.org gnome2-utils python-single-r1 vala
+
+DESCRIPTION="Input assistive technology intended for switch and pointer users"
+HOMEPAGE="https://wiki.gnome.org/Projects/Caribou"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+IUSE="gtk gtk2"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+KEYWORDS="~amd64 ~arm64 ~x86"
```

Comment:
Since Cinnamon is the only remaining in-tree consumer, and this is no longer being maintained by GNOME, I dropped the keywords that weren't also supported by Cinnamon. Is this ok, or should I add then back?

Reply:
1) You've absolutely done the right thing by asking, so thank you!

2) If it's possible to use Caribou stand-alone, please keep other keywords for now. Is it causing you any inconvenience (do we need to do some more keywording?), or was this just housekeeping?

Reply:
I expect caribou/antler probably can be used stand-alone. However, I don't have a system to test it on, since Cinnamon takes over the DBus interfaces.

That being said, from what I can find, `xvkbd` (in-tree) is the only OSK that has seen releases within the past year. `caribou` (in-tree), `eekboard` (in-tree), gok, onboard, and florence all seem to be abandoned/dead or on life support (i.e. compilation fixes only).

For some context, Up until #19082 upstream was years dead, and it's unused by GNOME. Fixes for that CVE regression did get merged upstream, but I doubt that's a commitment from upstream to continue maintaining it. https://github.com/linuxmint/cinnamon-screensaver/issues/354#issuecomment-759477865 suggests they might take over ownership, or fork. If that actually happens, I have some expectation they would probably strip it down to fit Cinnamon's needs.

From a practical standpoint, I only have amd64 systems, so I don't know what level of support I can provide to other arches. So this was mostly motivated by housekeeping.

Reply:
In that case, let's keep it for now. We tend to only drop things when they stop working or are causing a problem. We'll review depending on any issues which come up (and if they look arch-specific)?